* TOC
{:toc}

# WELCOME

Reliable detection of somatic variations is of critical importance in cancer research. Here we present [Lancet](https://github.com/omicsnut/v2_lancet), an accurate and sensitive somatic variant caller, which detects SNVs and indels by jointly analyzing reads from tumor and matched normal samples using colored de Bruijn graphs. 

# Getting Started
## Tutorials
## Examples
# Guides
## Testing
## Benchmarking
[Benchmarking Repo](https://github.com/nygenome/benchmark/tree/master/virtual_tumor/evaluation)

<a href="https://zhubry.github.io/LancetDocumentationExample/indel_20220528.performance-plots.pdf" target="_blank">Indel benchmark</a>

<a href="https://zhubry.github.io/LancetDocumentationExample/snv_20220528.performance-plots.pdf" target="_blank">SNV benchmark</a>
## Scripts
# Specs/Reference
## Memory Utilization
<a href="https://zhubry.github.io/LancetDocumentationExample/Memory_Utilized_vs_Length_SNV.pdf" target="_blank">SNV memory</a>

<a href="https://zhubry.github.io/LancetDocumentationExample/Memory_Utilized_vs_Length_indel.pdf" target="_blank">Indel memory</a>
## Threads/Cores and Runtime
<a href="https://zhubry.github.io/LancetDocumentationExample/Core_Hours_vs_Length_SNV.pdf" target="_blank">SNV runtime</a>

<a href="https://zhubry.github.io/LancetDocumentationExample/Core_Hours_vs_Length_indel.pdf" target="_blank">Indel runtime</a>
## Standards/references/docs
# Explanation
## This is Lancet
